src/app.js

```diff
@@ -1 +1,39 @@
+/* eslint-disable no-console */
 'use strict';
+
+const rl = require('readline');
+const { generateNumber } = require('./generateNumber');
+const { calculating } = require('./calculating');
+
+const number = generateNumber();
+
+const terminal = rl.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function playGame(startMessage) {
+  terminal.question(startMessage, (answer) => {
+    const { bulls, cows } = calculating(number, answer);
+
+    if (answer.length !== 4) {
+      console.log(`Should be a 4-digit number, you entered ${answer.length}\n`);
+      playGame('Try again: \n');
+
+      return;
+    }
+
+    console.log(`Bulls: ${bulls}, Cows: ${cows} \n`);
```

Comment:
Now the validation is going a bit wrong because it show `Cows` even though I haven't guessed any numbers. So the formatting itself is such that nothing is displayed for `Bulls`, it is worth setting the default `0` for these values.
(The first value is that I output the guessed number to the console.)
![image](https://github.com/mate-academy/node_bulls-and-cows/assets/54860641/4fb3ba38-39b0-484d-a34a-24a3bc4dbc1e)